paasta_tools/mesos_tools.py

```diff
@@ -86,12 +85,6 @@ def raise_cli_exception(msg):
 import mesos.cli.cluster  # noqa
 
 
-# monkey patch MesosMaster.state to use a larger ttl
-@util.CachedProperty(ttl=60)
```

Comment:
I would say leave this here for now. This is helping speed up *other* things that are not serviceinit. (like deploys, check scripts, etc)

I agree that we *can* remove this if you want to deploy the requests cache in all the places, but I'm ok if this PR just has serviceinit stuff first.